### ISSUE TITLE
Feature request: offer additional options for "Default button on Video/Audio OSD"

### ISSUE THREAD

**Issue by STPKITT** [b-jesch/skin.estuary.modv2#168]
Hi,
I personally would like to be able to set "Default button on Video/Audio OSD" to "Settings", but other people might desire other choices, so the skin should ideally give all possible options, although that would make it necessary to split that setting into two, one for Video and one for Audio.

**Comment by C4Wiz** [comment]
> Hi,
> 
> I personally would like to be able to set "Default button on Video/Audio OSD" to "Settings", but other people might desire other choices, so the skin should ideally give all possible options, although that would make it necessary to split that setting into two, one for Video and one for Audio.

Sounds like you need to fork the repo and make the changes you want. Thats what i did

**Comment by b-jesch** [comment]
I don't think so. It's confusing for some people to get different behaviours on video/audio OSD, especially when they use the remote control "blind" pressing the ok button.
